R/utils.R

```diff
@@ -148,3 +148,12 @@ restore_rng <- function(expr, seed = NULL) {
 
   expr
 }
+
+near <- function(x, y) {
+  tol <- .Machine$double.eps^0.5
+  abs(x - y) < tol
+}
+
+SW <- function(x) {
```

Comment:
For readability I prefer `suppressWarnings` rather than aliasing it as `SW`. It takes up more space, but future contributers don't need to dig around to find out what `SW` does.